dash/include/dash/memory/GlobHeapMem.h

```diff
@@ -429,6 +429,23 @@ class GlobHeapMem
     return unit_local_size;
   }
 
```

Comment:
Öhm actually the comment refers to the wrong line. What I mean is `GlobHeapMEm::distance`. And the problem is not the replicated logic itself. It is clear that `GlobHeapMem::distance` is different from that in `GlobStaticMem`. What I mean is the `gptr.addr_or_offset.offset / sizeof(element_type)` which is repeated in at least 3 or 4 file. We could provide a simple macro for this.